extensions/machine-learning/src/common/constants.ts

```diff
@@ -48,6 +49,7 @@ export const adsPythonBundleVersion = '0.0.1';
 export const msgYes = localize('msgYes', "Yes");
 export const msgNo = localize('msgNo', "No");
 export const managePackageCommandError = localize('mls.managePackages.error', "Package management is not supported for the server. Make sure you have Python or R installed.");
+export const verifyOdbcDriverError = localize('mls.verifyOdbcDriverError.error', "'{0}' is required  for package management. Please make sure it is installed and setup correctly.", supportedODBCDriver);
```

Comment:
"on server" right - lets have that clear in string?

Reply:
no locally


---
In reply to: [468935412](https://github.com/microsoft/azuredatastudio/pull/11587#discussion_r468935412) [](ancestors = 468935412)